components/Manual.tsx

```diff
@@ -495,7 +495,11 @@ function Manual() {
                     </svg>
                   </a>
                   <Markdown
-                    source={content.replace(/\$STD_VERSION/g, stdVersion)}
+                    source={
+                      content
+                        .replace(/\$STD_VERSION/g, stdVersion)
+                        .replace(/deno\/blob\/master/g, `deno/blob/v${version}`)
```

Comment:
I would rather we introduce a `$CLI_VERSION` string to replace, and use that in the manual. WDYT?

Reply:
I initially had thought about it after seeing `$STD_VERSION` in documentation - but this was an easier fix.

Reply:
Having said that, this fix will replace all links to master with `${version}` and may introduce bugs. Will replace `$CLI_VERSION` instead.

Reply:
Updated in [`b5cbb74` (#1528)](https://github.com/denoland/deno_website2/pull/1528/commits/b5cbb74e7e713d96993aa852554830cecbe478ca)